srcpkgs/folks/template

```diff
@@ -1,21 +1,21 @@
 # Template file for 'folks'
 pkgname=folks
-version=0.13.1
-revision=2
+version=0.14.0
+revision=1
 build_style=meson
 build_helper=gir
-configure_args="-Db_lto=false -Dlibsocialweb-backend=false"
-hostmakedepends="pkg-config intltool glib-devel vala python gobject-introspection
+configure_args="-Db_lto=false"
+hostmakedepends="pkg-config intltool glib-devel vala python3 gobject-introspection
```

Comment:
> During cross build it gives:
> 
> ```
> Program python3 (dbusmock) found: NO
> meson.build:135:2: ERROR: python3 is missing modules: dbusmock
> ```
> 
> So `python3-dbusmock` should probably be in `hostmakedepends` instead of `makedepends`, replacing `python3`.

Added.

Reply:
> Though this package will probably have to be nocross from what I see.

@mnabid Were you able to cross-compile it on your machine? 
Also `python3` can be removed from hostmakedepends as it's pulled in via `python3-dbusmock`.